app/styles/globals/_colors.scss

```diff
@@ -1,5 +1,5 @@
 :root {
-  --bg-color: hsl(0, 0%, 11%);
+  --bg-color: #202023;
   --text-color: hsl(0, 0%, 100%);
```

Comment:
why is this changed?

Reply:
In figma is that code color for background and nav and footer i make my nav and footer in color what is in figma and i see difference in colors and i check again and i see that are not same colors in figma and in _colors.scss